src/js/views/patients/sidebar/patient/patient-sidebar_views.js

```diff
@@ -0,0 +1,50 @@
+import Radio from 'backbone.radio';
+import { View } from 'marionette';
+
+import 'sass/modules/buttons.scss';
+import 'sass/modules/forms.scss';
+import 'sass/modules/textarea-flex.scss';
```

Comment:
I don't think you'll need `textarea-flex` or `forms`.

Reply:
I removed those. Should be included in the next commit.